src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -665,13 +672,13 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         Error(diagnostics, ErrorCode.ERR_AnonymousTypeDuplicatePropertyName, argumentSyntax.NameColon.Name);
```

Comment:
I think so, yes. One is about specifying all names (if you use any names), and the other is about reserved names ("Item1" and such).
@VSadov, what do you think?

I'll add test.